tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -305,3 +305,62 @@ rfc2S2N msg = mkAct recordType messageType writer
                  | msg.sender == both then 'B'
                  else '!'
         noMessageType = TLS_HELLO_REQUEST // HelloRequests don't exist in tls1.3
+
+/**
+ * printHandshake and s2nToWords are two helper functions for debugging cryptol
+ * Take the counterexample from tls13rfcSimulatesS2N and plug the values into
+ * testParameters and testConnection. Run printHandshake with ascii set to on. 
+ * Note with ascii on, you may see actual_protocol_version = '"'. This is because
+ * decimal value 34 (hex value 22) == asci character double quotes. 
+ */
+
+testParameters : Parameters
+testParameters = {
+            compat_mode = [False, True],
```

Comment:
Do we want to set all of these to some consistent defaults just for code cleanliness? False, 0, etc?

Reply:
sure!